pages/api/views/[slug].js

```diff
@@ -1,25 +1,44 @@
-import db from '@/lib/firebase';
+import db from '@/lib/planetscale';
 
 export default async function handler(req, res) {
-  if (req.method === 'POST') {
-    const ref = db.ref('views').child(req.query.slug);
-    const { snapshot } = await ref.transaction((currentViews) => {
-      if (currentViews === null) {
-        return 1;
+  try {
+    if (req.method === 'POST') {
+      const [result] = await db.query(
+        `
+        INSERT INTO views (slug) VALUES(?)
+        ON DUPLICATE KEY
+        UPDATE count = last_insert_id(count + 1), updated_at = now();
```

Comment:
Hi @leerob 

From the latest preview URL this errors out with 500 status code, to reproduce just open the network tab and visit any blog, the error returned from planetscale client complaining that you don't have `updated_at` column. So the views will never increment. I suggest to wrap all Planetscale functions in a try catch block, so the errors doesn't leak info about your db instance. Also this might occur in other places where you have `updated_at`.  

Another thing to note this piece of code:
```updated_at = now();```
is setting the updated_at to the current time. The current time in this context using MySQL `now()` function will depend on Planetscale database instance time zone. Which might differ than what you intend to do. The previous version was setting the current date using the JavaScript function  
```js
updated_at: Date.now()
```
and I suppose this would be the same as your server time on Vercel?
This would apply to other places in the code as well.

Reply:
I noticed! Thanks for the info 😄 Will be fixing this shortly.